net-utils/Cargo.toml

```diff
@@ -11,30 +11,38 @@ edition = { workspace = true }
 
 [dependencies]
 bincode = { workspace = true }
-clap = { version = "3.1.5", features = ["cargo"] }
+clap = { version = "3.1.5", features = ["cargo"], optional = true }
 crossbeam-channel = { workspace = true }
 log = { workspace = true }
 nix = { workspace = true, features = ["socket"] }
 rand = { workspace = true }
 serde = { workspace = true }
 serde_derive = { workspace = true }
 socket2 = { workspace = true }
-solana-logger = { workspace = true }
 solana-sdk = { workspace = true }
-solana-version = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 url = { workspace = true }
 
+[dev-dependencies]
+solana-logger = { workspace = true }
+solana-version = { workspace = true }
```

Comment:
It looks like `solana-logger` may be a dev-dependency, but `solana-version` is required for the two CLIs, so it needs to be an optional dependency for the new `clap` feature, at a minimum. Alternatively you can just leave these where they were under dependencies.